### DISCUSSION TITLE
Service discovery for RoadRunner

### DISCUSSION THREAD

**Discussion by lanphan** [roadrunner-server/roadrunner#1039] [Q&A] [answered]
Hello,

Do we have plan to support Service discovery feature?
* Register Roadrunner when start (with etcd / consul / cloud providers ...)
* Fetch and cache data from Service discovery, so that if etcd die, it still have old data to connect

**Comment by rustatian** [comment, marked as answer]
@lanphan Hey, yeah, we have a ticket for that https://github.com/roadrunner-server/roadrunner/issues/915

**Reply by lanphan** [reply]
thank @rustatian 
May I know that do you have some guess about when that ticket will be released?

**Reply by rustatian** [reply]
More likes - sooner release 🙂The formulae is simple 😉

**Reply by rustatian** [reply]
You can tell people about that FR in your community or in our discord. Discuss it, so, more people involved, sooner it will be released.
